package.json

```
@@ -1,67 +1,69 @@
 {
   "name": "warcio",
-  "version": "1.6.0",
-  "main": "index.js",
-  "type": "module",
-  "license": "Apache-2.0",
-  "author": "Webrecorder Software, Ilya Kreymer",
-  "bin": {
-    "warcio.js": "./cli.js"
-  },
-  "exports": {
-    ".": "./index.js"
-  },
+  "version": "2.0.0",
+  "keywords": [
+    "WARC",
+    "web archiving"
+  ],
   "homepage": "https://github.com/webrecorder/warcio.js#readme",
+  "bugs": {
+    "url": "https://github.com/webrecorder/warcio.js/issues"
+  },
   "repository": {
     "type": "git",
     "url": "https://github.com/webrecorder/warcio.js.git"
   },
-  "bugs": {
-    "url": "https://github.com/webrecorder/warcio.js/issues"
-  },
-  "keywords": [
-    "WARC",
-    "web archiving"
+  "license": "Apache-2.0",
+  "author": "Webrecorder Software, Ilya Kreymer",
+  "sideEffects": [
+    "./src/polyfills.cjs"
   ],
-  "engines": {
-    "node": ">=18.0.0"
+  "type": "module",
+  "exports": {
+    "require": "./dist/index.cjs",
+    "default": "./dist/index.js"
   },
+  "main": "dist/index.cjs",
+  "module": "dist/index.js",
+  "source": "src/index.ts",
+  "types": "dist/index.d.ts",
+  "bin": "./dist/cli.js",
```

It looks like this needs to be the .cjs version for the cli command to work, without the dynamic import

```suggestion
  "bin": "./dist/cli.cjs",
```